lambda/service/handler/get_pennsieve_accounts_handler.go

```diff
@@ -0,0 +1,46 @@
+package handler
+
+import (
+	"encoding/json"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/pennsieve/account-service/service/models"
+	"github.com/pennsieve/account-service/service/utils"
+)
+
+const (
+	AWS = "aws"
+)
+
+func GetPennsieveAccountsHandler(request events.APIGatewayV2HTTPRequest) (events.APIGatewayV2HTTPResponse, error) {
+	handlerName := "GetPennsieveAccountsHandler"
+	accountType := utils.ExtractParam(request.RawPath)
+	accountId := request.RequestContext.AccountID
```

Comment:
resorted to testing the above first, will update to https://github.com/aws/aws-sdk-go-v2/issues/214#issuecomment-1543252691, if the above implementation does not work.